ERROR - 2023-08-30 15:55:40 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost:3307', 'root', '', 'sastreriajimene...', 3307, '', 0)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(11): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->index()
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(11): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->index()
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-30 15:55:40 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `libros`
ORDER BY `id` ASC', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Libros.php(11): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 15:58:24 --> Undefined array key "apellido"
in APPPATH\Views\libros\listar.php on line 33.
 1 APPPATH\Views\libros\listar.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "apellido"', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php', 33)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Libros.php(16): view('libros/listar', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-30 18:05:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'biblioteca', 3306, '', 0)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(11): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->index()
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(11): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->index()
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-30 18:05:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `libros`
ORDER BY `id` ASC', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Libros.php(11): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-30 18:06:16 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(56): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->guardar()
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-30 18:06:16 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'DAVID\', \'ramos\', \'77448360\', \'1693418776_6b372614e7036319dc75.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'DAVID\', \'ramos\', \'77448360\', \'1693418776_6b372614e7036319dc75.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'DAVID\', \'ramos\', \'77448360\', \'1693418776_6b372614e7036319dc75.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (:nombre:, :apellido:, :celular:, :imagen:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(56): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-30 18:13:51 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(56): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->guardar()
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-30 18:13:51 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'DAVID\', \'ramos\', \'77448360\', \'1693419231_5144932f573776dd736b.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'DAVID\', \'ramos\', \'77448360\', \'1693419231_5144932f573776dd736b.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'DAVID\', \'ramos\', \'77448360\', \'1693419231_5144932f573776dd736b.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (:nombre:, :apellido:, :celular:, :imagen:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(56): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-30 18:14:11 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(56): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->guardar()
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-30 18:14:11 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'juan\', \'ramos\', \'77448360\', \'1693419251_5a9de9161459cab45e3b.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'juan\', \'ramos\', \'77448360\', \'1693419251_5a9de9161459cab45e3b.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'juan\', \'ramos\', \'77448360\', \'1693419251_5a9de9161459cab45e3b.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (:nombre:, :apellido:, :celular:, :imagen:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(56): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-30 18:14:22 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(56): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->guardar()
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-30 18:14:22 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'juan\', \'ramos\', \'77448360\', \'1693419262_43ce64411e6e991a5986.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'juan\', \'ramos\', \'77448360\', \'1693419262_43ce64411e6e991a5986.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'juan\', \'ramos\', \'77448360\', \'1693419262_43ce64411e6e991a5986.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (:nombre:, :apellido:, :celular:, :imagen:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(56): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-30 18:15:17 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(56): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->guardar()
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-30 18:15:17 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'asasa\', \'sasas\', \'asasa\', \'1693419317_e0bb489598eb4536a760.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'asasa\', \'sasas\', \'asasa\', \'1693419317_e0bb489598eb4536a760.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'asasa\', \'sasas\', \'asasa\', \'1693419317_e0bb489598eb4536a760.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (:nombre:, :apellido:, :celular:, :imagen:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(56): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-30 18:17:39 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\app\Controllers\Libros.php(56): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(934): App\Controllers\Libros->guardar()
#9 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CodeNaiter4\RamosJhonathan\biblioteca\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-30 18:17:39 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'jhonathan david\', \'ramos jimenez\', \'77448360\', \'1693419459_70adec5332a9fb99d942.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'jhonathan david\', \'ramos jimenez\', \'77448360\', \'1693419459_70adec5332a9fb99d942.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (\'jhonathan david\', \'ramos jimenez\', \'77448360\', \'1693419459_70adec5332a9fb99d942.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`nombre`, `apellido`, `celular`, `imagen`) VALUES (:nombre:, :apellido:, :celular:, :imagen:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(56): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 18:18:17 --> Undefined array key "nombre"
in APPPATH\Views\libros\listar.php on line 32.
 1 APPPATH\Views\libros\listar.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombre"', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php', 32)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Libros.php(16): view('libros/listar', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 18:18:21 --> Undefined array key "nombre"
in APPPATH\Views\libros\listar.php on line 32.
 1 APPPATH\Views\libros\listar.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombre"', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php', 32)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Libros.php(16): view('libros/listar', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 18:18:21 --> Undefined array key "nombre"
in APPPATH\Views\libros\listar.php on line 32.
 1 APPPATH\Views\libros\listar.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombre"', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php', 32)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Libros.php(16): view('libros/listar', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 18:18:22 --> Undefined array key "nombre"
in APPPATH\Views\libros\listar.php on line 32.
 1 APPPATH\Views\libros\listar.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombre"', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php', 32)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Libros.php(16): view('libros/listar', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 18:20:40 --> unlink(../public/uploads/imagen.jpg): No such file or directory
in APPPATH\Controllers\Libros.php on line 70.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(../public/uploads/imagen.jpg): No such file or directory', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Controllers\\Libros.php', 70)
 2 APPPATH\Controllers\Libros.php(70): unlink('../public/uploads/imagen.jpg')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->borrar('1')
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 18:21:02 --> Trying to access array offset on value of type null
in APPPATH\Controllers\Libros.php on line 69.
 1 APPPATH\Controllers\Libros.php(69): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Controllers\\Libros.php', 69)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->borrar('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 18:21:03 --> Trying to access array offset on value of type null
in APPPATH\Controllers\Libros.php on line 69.
 1 APPPATH\Controllers\Libros.php(69): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Controllers\\Libros.php', 69)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->borrar('1')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:09:35 --> include(header.php): Failed to open stream: No such file or directory
in APPPATH\Views\libros\login.php on line 1.
 1 APPPATH\Views\libros\login.php(1): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(header.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\login.php', 1)
 2 APPPATH\Views\libros\login.php(1): include()
 3 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\login.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/login', [], true)
 6 APPPATH\Controllers\Libros.php(33): view('libros/login', [...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->login()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:12:52 --> Invalid file: "libros/login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('libros/login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/login', [], true)
 3 APPPATH\Controllers\Libros.php(33): view('libros/login', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:12:53 --> Invalid file: "libros/login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('libros/login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/login', [], true)
 3 APPPATH\Controllers\Libros.php(33): view('libros/login', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:12:53 --> Invalid file: "libros/login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('libros/login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/login', [], true)
 3 APPPATH\Controllers\Libros.php(33): view('libros/login', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:12:59 --> Invalid file: "libros/login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('libros/login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/login', [], true)
 3 APPPATH\Controllers\Libros.php(33): view('libros/login', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:14:57 --> Invalid file: "usuarios/login2.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/login2.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/login2', [], true)
 3 APPPATH\Controllers\Usuarios.php(11): view('usuarios/login2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:14:58 --> Invalid file: "usuarios/login2.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/login2.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/login2', [], true)
 3 APPPATH\Controllers\Usuarios.php(11): view('usuarios/login2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:14:58 --> Invalid file: "usuarios/login2.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/login2.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/login2', [], true)
 3 APPPATH\Controllers\Usuarios.php(11): view('usuarios/login2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:14:58 --> Invalid file: "usuarios/login2.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/login2.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/login2', [], true)
 3 APPPATH\Controllers\Usuarios.php(11): view('usuarios/login2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:14:58 --> Invalid file: "usuarios/login2.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/login2.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/login2', [], true)
 3 APPPATH\Controllers\Usuarios.php(11): view('usuarios/login2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:14:59 --> Invalid file: "usuarios/login2.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/login2.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/login2', [], true)
 3 APPPATH\Controllers\Usuarios.php(11): view('usuarios/login2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:14:59 --> Invalid file: "usuarios/login2.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/login2.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/login2', [], true)
 3 APPPATH\Controllers\Usuarios.php(11): view('usuarios/login2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:14:59 --> Invalid file: "usuarios/login2.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/login2.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/login2', [], true)
 3 APPPATH\Controllers\Usuarios.php(11): view('usuarios/login2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:15:05 --> Invalid file: "usuarios/login2.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/login2.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/login2', [], true)
 3 APPPATH\Controllers\Usuarios.php(11): view('usuarios/login2')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:17:46 --> Invalid file: "libros/login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('libros/login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/login', [], true)
 3 APPPATH\Controllers\Libros.php(33): view('libros/login', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:17:47 --> Invalid file: "libros/login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('libros/login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/login', [], true)
 3 APPPATH\Controllers\Libros.php(33): view('libros/login', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:17:47 --> Invalid file: "libros/login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('libros/login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/login', [], true)
 3 APPPATH\Controllers\Libros.php(33): view('libros/login', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:17:48 --> Invalid file: "libros/login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('libros/login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/login', [], true)
 3 APPPATH\Controllers\Libros.php(33): view('libros/login', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Libros->login()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:27:56 --> Invalid file: "usuarios/register.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/register.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuarios/register', [], true)
 3 APPPATH\Controllers\Usuarios.php(16): view('usuarios/register')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->register()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:30:00 --> include(header.php): Failed to open stream: No such file or directory
in APPPATH\Views\libros\register.php on line 1.
 1 APPPATH\Views\libros\register.php(1): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(header.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\register.php', 1)
 2 APPPATH\Views\libros\register.php(1): include()
 3 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\register.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/register', [], true)
 6 APPPATH\Controllers\Usuarios.php(16): view('libros/register')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Usuarios->register()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:48:23 --> Invalid file: "index/listar.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('index/listar.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('index/listar', [], true)
 3 APPPATH\Controllers\Home.php(9): view('index/listar')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:48:51 --> Undefined variable $cabecera
in APPPATH\Views\libros\listar.php on line 1.
 1 APPPATH\Views\libros\listar.php(1): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $cabecera', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php', 1)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Home.php(9): view('libros/listar')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:48:57 --> Undefined variable $cabecera
in APPPATH\Views\libros\listar.php on line 1.
 1 APPPATH\Views\libros\listar.php(1): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $cabecera', 'C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php', 1)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\CodeNaiter4\\RamosJhonathan\\biblioteca\\app\\Views\\libros\\listar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('libros/listar', [], true)
 5 APPPATH\Controllers\Home.php(9): view('libros/listar')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 21:56:30 --> Invalid file: "index.html"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('index.html')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('index.html', [], true)
 3 APPPATH\Controllers\Home.php(9): view('index.html')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 22:00:50 --> Invalid file: "index.html"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('index.html')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('index.html', [], true)
 3 APPPATH\Controllers\Home.php(9): view('index.html')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-30 22:00:55 --> Invalid file: "index.html"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('index.html')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('index.html', [], true)
 3 APPPATH\Controllers\Home.php(9): view('index.html')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
